terraform/README.md

```
@@ -0,0 +1,140 @@
+
+```bash
+terraform apply
+```
+
+## 新しく環境を追加する場合（ステージング環境など）
+
+**Owner権限が必要**
+
+env以下にルートモジュールを追加する。
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)